src/serializer/MapFishPrintV3XYZSerializer.ts

```diff
@@ -0,0 +1,56 @@
+import OlSource from 'ol/source/Source';
+import OlSourceXYZ from 'ol/source/XYZ';
+import OlLayer from 'ol/layer/Layer';
+
+import BaseSerializer from './BaseSerializer';
+
+export class MapFishPrintV3XYZSerializer implements BaseSerializer {
+
+  /**
+   * The XYZ layer type identificator.
+   * We use OSM for the type because the backend does not
+   * have XYZ implemented which leads to the map not being printed
+   */
+  static TYPE_XYZ: string = 'OSM';
+
+  validateSource(source: OlSource): source is OlSourceXYZ {
+    return source instanceof OlSourceXYZ;
+  }
+
+  serialize(olLayer: OlLayer, opts?: any) {
+    const optsToApply = {
+      baseURL: 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
+      customParams: {},
+      dpi: 72,
+      failOnError: false,
+      imageExtension: 'png',
+      maxExtent: [-20037508.34, -20037508.34, 20037508.34, 20037508.34],
+      rasterStyle: undefined,
+      tileSize: [256, 256],
```

Comment:
I think these props are specific to the OlSourceOSM, so they should not be used here

Reply:
Which ones exactly?

Reply:
https://openlayers.org/en/latest/apidoc/module-ol_source_OSM-OSM.html might help

Reply:
done in: [83c1cb4](https://github.com/terrestris/mapfish-print-manager/pull/482/commits/83c1cb4080895eb0c35207f2e6116255a1bcc31e)